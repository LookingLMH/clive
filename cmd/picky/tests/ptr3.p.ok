stack trace at:
main() pid 2	pc 0x00000038 ptr3.p:28
	local variables:
	p = 0x2103173c0
	q = 0x2103173c0

global variables:
Maxint = 2147483647
Minint = -2147483646
Maxchar = '每'
Minchar = '\000'
Eol = \n
Eof = '每'
Tab = \t
Esc = '\033'
Nul = '\000'
stdin = <stdin>
stdout = <stdout>
heap:
stack trace at:
main() pid 2	pc 0x00000046 ptr3.p:32
	local variables:
	p = 0x2103173c0
	q = 0x2103173c0

global variables:
Maxint = 2147483647
Minint = -2147483646
Maxchar = '每'
Minchar = '\000'
Eol = \n
Eof = '每'
Tab = \t
Esc = '\033'
Nul = '\000'
stdin = <stdin>
stdout = <stdout>
heap:
3
