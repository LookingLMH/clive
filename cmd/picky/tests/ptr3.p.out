stack trace at:
main() pid 2	pc 0x00000038 ptr3.p:28
	local variables:
	p = 0xc208040100
	q = 0xc208040100

global variables:
Maxint = 2147483647
Minint = -2147483646
Maxchar = 'ÿ'
Minchar = '\000'
Minstrength = %!v(PANIC=runtime error: index out of range)
Maxstrength = %!v(PANIC=runtime error: index out of range)
Transp = 0.0
Tlucid = 0.5
Opaque = 1.0
NoBut = %!v(PANIC=vfmt: fmt 75)
Shift = 'ñ'
MetaLeft = '÷'
Eol = \n
Down = 'ô'
Left = 'ó'
Ctrl = 'ð'
Del = 'ù'
MetaRight = 'ø'
Esc = '\033'
Up = 'õ'
Return = 'ö'
Eof = 'ÿ'
Right = 'ò'
Tab = \t
Nul = '\000'
stdin = <stdin>
stdout = <stdout>
stdgraph = invalid file
heap:
0xc208040100 = 3
stack trace at:
main() pid 2	pc 0x00000046 ptr3.p:32
	local variables:
	p = 0xc208040100
	q = 0xc208040100

global variables:
Maxint = 2147483647
Minint = -2147483646
Maxchar = 'ÿ'
Minchar = '\000'
Minstrength = %!v(PANIC=runtime error: index out of range)
Maxstrength = %!v(PANIC=runtime error: index out of range)
Transp = 0.0
Tlucid = 0.5
Opaque = 1.0
NoBut = %!v(PANIC=vfmt: fmt 75)
Shift = 'ñ'
MetaLeft = '÷'
Eol = \n
Down = 'ô'
Left = 'ó'
Ctrl = 'ð'
Del = 'ù'
MetaRight = 'ø'
Esc = '\033'
Up = 'õ'
Return = 'ö'
Eof = 'ÿ'
Right = 'ò'
Tab = \t
Nul = '\000'
stdin = <stdin>
stdout = <stdout>
stdgraph = invalid file
heap:
3
